src/api/validators/ExperimentConditionValidator.ts

```diff
@@ -0,0 +1,14 @@
+import { IsDefined, IsNotEmpty } from 'class-validator';
```

Comment:
This seems same as ExperimentCondition model can we use that class itself for validating instead of creating a duplicate class here?

Reply:
Yeah! i used models classes for all validations.

Reply:
I mean instead of creating validators/* can we use models itself for validating?